### DISCUSSION TITLE
[Feature] Share all photos of an individual

### DISCUSSION THREAD

**Discussion by Tuporat** [immich-app/immich#11107] [Feature Request]
### I have searched the existing feature requests to make sure this is not a duplicate request.

- [X] Yes

### The feature

Hi, I would like to be able to share all recognized photos of a person directly with them, as long as they are a registered user of the Immich instance. As if the person was an album, so to speak. 
Best regards and many thanks for the great work. Immich brings my photo library to life.

### Platform

- [X] Server
- [x] Web
- [x] Mobile

**Comment by radh21301** [comment]
This is a bigger feature request that also includes your use case: https://github.com/immich-app/immich/discussions/1673
